src/frontend/src/pages/index.js

```diff
@@ -40,12 +42,67 @@ export default function IndexPage() {
     getPosts(numPages);
   }, [telescopeUrl]);
 
+  async function getPostsCount() {
+    try {
+      const res = await fetch(`${telescopeUrl}/posts`, { method: 'HEAD' });
+      if (!res.ok) {
+        throw new Error(res.statusText);
+      }
+      return res.headers.get('x-total-count');
+    } catch (error) {
+      console.log(error);
+    }
+    return null;
+  }
+
+  function callback() {
+    getPostsCount()
+      .then(setCurrentNumPosts)
+      .catch((error) => console.log(error));
+  }
+
+  useEffect(() => {
+    savedCallback.current = callback;
+  });
+
+  // Get the current + initial posts count when page loads
+  useEffect(() => {
+    async function setPostsInfo() {
+      try {
+        await Promise.all([
+          setInitNumPosts(await getPostsCount()),
+          setCurrentNumPosts(await getPostsCount()),
+        ]);
+      } catch (error) {
+        console.log({ error });
+      }
+    }
+    setPostsInfo();
+  }, []);
+
+  useEffect(() => {
+    function getCurrentNumPosts() {
+      savedCallback.current();
+      console.log(currentNumPosts);
+    }
+
+    savedCallback.current = callback;
+    // Polls every 5 minute
+    const interval = setInterval(getCurrentNumPosts, 300000);
```

Comment:
Instead of this, do `5 * 60 * 1000` so that it's more obvious how you arrived at it.